### PR TITLE
One Member Relation Check

### PR DESCRIPTION
**Pull request by savannahostrowski**
The OneMemberRelation Check flags Relations that only have one Member. 

This pull request contains logic, documentation, configuration, and unit tests.

This PR includes flag logic, documentation, configuration, and unit tests.

**OSM Examples:**
https://www.openstreetmap.org/relation/6077987
https://www.openstreetmap.org/relation/7214599

### CODE REVIEW

**Comment by savannahostrowski** [comment]
@kkonishi2 If you have anything else to document for other enhancements, please write them here so I can keep track! 😄

**Comment by savannahostrowski** [comment]
@danielduhh Ready for review.

File: src/test/java/org/openstreetmap/atlas/checks/validation/relations/OneMemberRelationCheckTestRule.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            relations = { @Relation(id = "123", members = {
+                    @Member(id = "12", type = "edge", role = RelationTypeTag.MULTIPOLYGON_ROLE_INNER),
+                    @Member(id = "2", type = "node", role = RelationTypeTag.MULTIPOLYGON_ROLE_OUTER),
+                    @Member(id = "23", type = "edge", role = RelationTypeTag.MULTIPOLYGON_ROLE_OUTER) }, tags = {
+                            "restriction=no_u_turn", "type=multipolygon" }) })
+    private Atlas validRelationMultipolygon;
+
+    public Atlas getValidRelation()
+    {
+        return this.validRelation;
```
Comment (danielduhh) [inline src/test/java/org/openstreetmap/atlas/checks/validation/relations/OneMemberRelationCheckTestRule.java]: nit: `this.` is used for the first two getters; For consistency sake, we should choose one way or another

File: docs/checks/oneMemberRelationCheck.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+#### Description
+
+This check identifies relations that only contain one member. In OSM, a [Relation](https://wiki.openstreetmap.org/wiki/Elements#Relation)
+is a multi-purpose data structure that documents a relationship between two or more data elements 
+(nodes, ways, and/or other relations).
+
+#### Live Example
+The following examples illustrate two cases where a Relation contains only one member.
+1) This relation [id:4646212](https://www.openstreetmap.org/relation/4646212) has only one member.
+2) This relation [id:7691824](ttps://www.openstreetmap.org/relation/7691824) has only one member.
```
Comment (danielduhh) [inline docs/checks/oneMemberRelationCheck.md]: missing the `h` in https

**Comment by danielduhh** [comment]
LGTM

**Comment by danielduhh** [comment]
LIF (let it fly)

**Pull request closed without merging**